test/unit/sort-locales.test.js

```diff
@@ -0,0 +1,99 @@
+import sortLocales from '../../lib/utils/sort-locales'
+
+/*
+Those locales are dependent like this:
+
+`base` and `base2`    -> don't have a fallback
+`a`, `b`, `c`         -> have `base` as a fallback
+`fallback_a`          -> falls back to `a`
+`fallback_fallback_a` -> falls back to `fallback_a`
+`fallback_b`          -> falls back to `b`
+
+base < a < fallback_a < fallback_fallback_a
+base < b < fallback_b
+base < c < fallback_c
+base2
+ */
+const locales = [
+  {
+    code: 'fallback_fallback_a',
+    fallbackCode: 'fallback_a'
+  },
+  {
+    code: 'base',
+    fallbackCode: null
+  },
+  {
+    code: 'a',
+    fallbackCode: 'base'
+  },
+  {
+    code: 'b',
+    fallbackCode: 'base'
+  },
+  {
+    code: 'fallback_a',
+    fallbackCode: 'a'
+  },
+  {
+    code: 'fallback_c',
+    fallbackCode: 'c'
+  },
+  {
+    code: 'base2',
+    fallbackCode: null
+  },
+  {
+    code: 'fallback_b',
+    fallbackCode: 'b'
+  },
+  {
+    code: 'c',
+    fallbackCode: 'base'
+  }
+]
+
+test('Sorts locales by "fallback" order', () => {
+  const sortedEntries = sortLocales(locales)
```

Comment:
Did you mean `sortedLocales` instead of `sortedEntries`?

Reply:
Yeah, that looks like a copy'n'past error, thanks for spotting.